cheddar/cheddar-integration-aws/src/test/java/com/clicktravel/infrastructure/persistence/aws/cloudsearch/CloudSearchEngineTest.java

```diff
@@ -770,9 +732,33 @@ public void shouldSearch_withDefaultSortInQuery() throws Exception {
     @Test
     public void shouldNotSearch_withNullSearchOptions() throws Exception {
         // Given
+        final StubDocument document = randomStubDocument();
```

Comment:
I can see that the `cacheDomainEndpoints()` method causes this complexity in the test. It hints that we should really extract this method into its own class... but I'm happy with this as it is to stick with the original scope of this PR.